cmake/protobuf-targets.cmake

```diff
@@ -0,0 +1,17 @@
+# Library aliases
+foreach(_library

```

Comment:
No for build tree configuration. Needed for build examples in main process (if ptotobuf_BUILD_EXAMPLES enabled).